<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lookoverImage</class>
 <widget class="QDialog" name="lookoverImage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="beforePushButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>340</y>
     <width>45</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>before</string>
   </property>
  </widget>
  <widget class="QPushButton" name="afterPushButton">
   <property name="geometry">
    <rect>
     <x>203</x>
     <y>340</y>
     <width>45</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>after</string>
   </property>
  </widget>
  <widget class="QLabel" name="numberLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>340</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>number</string>
   </property>
  </widget>
  <widget class="QLabel" name="imageLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
   <property name="text">
    <string>imageLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delPushButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>340</y>
     <width>45</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>del</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
